### DISCUSSION TITLE
阅读限制pro 插件 记录问题

### DISCUSSION THREAD

**Discussion by wxyShine** [halo-dev/halo#6931] [Q&A]
### 系统信息

- 外部访问地址: https://www.wxy97.com
- 启动时间: 2024-10-22 17:19
- 版本: 2.20.6
- 构建时间: 2024-10-22 16:19
- Git Commit: 1436a9e
- Java: IBM Semeru Runtime Open Edition / 21.0.4+7-LTS
- 数据库: PostgreSQL / 16.4 (Debian 16.4-1.pgdg120+2)
- 操作系统: Linux / 6.1.0-10-amd64
- 已激活主题: [Walker 1.12.1](https://www.halo.run/store/apps/app-GHgAR)
- 已启动插件:
  - [内容格式助手 1.2.0](https://github.com/ruibaby/plugin-content-tools)
  - [lightgallery.js 灯箱 1.2.1](https://github.com/halo-sigs/plugin-lightgallery)
  - [友链自助提交插件 2.0.4](https://github.com/liuyiwuqing/halo-plugin-links-submit)
  - [链上好文 1.0.1](https://github.com/guqing/plugin-share-post-via-link)
  - [编辑器超链接卡片 1.1.0](https://github.com/halo-sigs/plugin-editor-hyperlink-card)
  - [Docsme 1.0.0-alpha.8](https://www.halo.run/store/apps/app-yffxw)
  - [豆瓣 1.2.1](https://github.com/chengzhongxue/plugin-douban)
  - [Data Studio（数据工厂） 1.0.0-alpha.2](https://github.com/halo-sigs/plugin-data-studio)
  - 联系表单 1.0.2
  - [Umami 1.4.0](https://github.com/halo-sigs/plugin-umami)
  - [highlight.js 代码高亮 1.3.2](https://github.com/halo-sigs/plugin-highlightjs)
  - [静态网页服务 1.0.0-alpha.5](https://www.halo.run/store/apps/app-gFkMn)
  - [OAuth2 认证 1.4.1](https://github.com/halo-sigs/plugin-oauth2)
  - [SEO 工具集 1.1.0](https://www.halo.run/store/apps/app-FNGbT)
  - [Vditor 编辑器 1.8.0](https://github.com/justice2001/halo-plugin-vditor)
  - [文本绘图 1.3.0](https://github.com/halo-sigs/plugin-text-diagram)
  - [Meilisearch 2.0.1](https://github.com/Rainsheep/halo-plugin-meilisearch)
  - [AI 助手 1.0.0](https://www.halo.run/store/apps/app-riNgb)
  - [增强备份（Backup IMproved） 1.1.1](https://www.halo.run/store/apps/app-dHakX)
  - [Metrics Graph 1.1.0](https://github.com/guqing/plugin-metrics-graph)
  - [Markdown / HTML 内容块 1.2.0](https://github.com/halo-dev/plugin-hybrid-edit-block)
  - [文章限制阅读 Pro 1.4.2](https://www.halo.run/store/apps/app-TzRqd)
  - [RSS 1.3.0](https://github.com/halo-dev/plugin-feed)
  - [Sitemap 1.1.2](https://github.com/halo-dev/plugin-sitemap)
  - [搜索组件 1.6.2](https://github.com/halo-dev/plugin-search-widget)
  - [评论组件 2.5.0](https://github.com/halo-dev/plugin-comment-widget)
  - [应用市场 1.7.0](https://www.halo.run/store/apps/app-VYJbF)
  - [瞬间 1.8.0](https://github.com/halo-sigs/plugin-moments)
  - [链接管理 1.5.0](https://github.com/halo-sigs/plugin-links)
  - [图库管理 1.4.0](https://github.com/halo-sigs/plugin-photos)
  - [对象存储（Amazon S3 协议） 1.12.0](https://github.com/halo-dev/plugin-s3)

### 使用的哪种方式运行？

Docker

### 发生了什么？

![Image](https://github.com/user-attachments/assets/fb212b5b-6d97-47bd-80b4-d1f2a60147d7)

![Image](https://github.com/user-attachments/assets/bf69c213-8160-433b-a1c3-a820d7520f21)

![Image](https://github.com/user-attachments/assets/76f815cc-d176-4266-a425-2cecae42bd7c)

10-23号已验证的码 记录里面查不到

还有10-22 已经验证过的消息码 在记录里面也找不到对应的码

**Comment by ruibaby** [comment]
cc @liuchangfitcloud

**Comment by Arch4AI** [comment]
验证通过但是没有保存记录吗？

**Reply by wxyShine** [reply]
是的。记录排序 和记录刷新都有点问题应该 
默认最新是26号。选择排序后最新是22( 也不是，每次都毫无规律的变化好像)
![image](https://github.com/user-attachments/assets/9cf39f14-7325-44a6-9ed8-b98ff427cac1)
![image](https://github.com/user-attachments/assets/c745b813-c8f6-42a1-81ea-0f3a89a71670)


![image](https://github.com/user-attachments/assets/90776174-674a-47c5-9eb2-00ec94fac64b)

**Reply by Arch4AI** [reply]
我需要重现一下。

**Reply by wxyShine** [reply]
好的 可以复现吗~  我这是每次都能复现。

**Reply by Arch4AI** [reply]
查询的问题复现了。但是生成记录我这边是正常的。

**Reply by Arch4AI** [reply]
如果是正常无法获取到码的时候是否可以提供对应的日志信息给我。

**Reply by wxyShine** [reply]
> 查询的问题复现了。但是生成记录我这边是正常的。

那估计是数据太多。排序问题导致最新的记录我无法看到。

**Reply by Arch4AI** [reply]
好的，今天我发一个rc你到时更新看看

**Reply by wxyShine** [reply]
测试了下新版 没问题了

**Reply by Arch4AI** [reply]
好的
